reference/tools/user-guide.md

```diff
@@ -6,184 +6,270 @@ aliases: ['/docs/dev/how-to/migrate/from-mysql/','/docs/dev/how-to/migrate/incre
+#### Mydumper
 
-[Mydumper](/reference/tools/mydumper.md) is a MySQL community tool used for full logical backups of MySQL that also works with TiDB.
+[Mydumper](/reference/tools/mydumper.md) is a tool to create a logical full backup for TiDB.
 
 The following are the basics of Mydumper:
 
-- Input: MySQL/TiDB clusters.
-- Output: SQL files.
-- Compatibility: Compatible with all TiDB versions.
-- Kubernetes: Supported. See [Backup and Restore](/tidb-in-kubernetes/maintain/backup-and-restore.md).
+- Input/Output
+    - Input: TiDB/MySQL host:port 
+    - Output: schema and insert statements file
+- Supported TiDB versions: all versions
+- Kubernetes support: Yes. See [Backup and Restore](/tidb-in-kubernetes/maintain/backup-and-restore.md) for details.
 
-#### Full export tool TiDB Binlog
+#### BR (beta)
 
-[TiDB Binlog](/reference/tidb-binlog/overview.md) is a tool used to collect binlog data from TiDB. It provides near real-time backup and replication to downstream platforms.
+[BR](/reference/tools/br/br.md) (Backup & Restore) is a command-line tool for distributed backup and restoration of the TiDB cluster data. Compared with Mydumper/loader, BR is more suitable for scenarios of huge data volume.
 
-The following are the basics of TiDB Binlog:
+The following are the basics of BR:
+
+- Input/Output
+    - Input: TiDB cluster
+    - Output: Full backup file
+- Supported TiDB versions: v3.1 or v4.0
+- Kubernetes support: Yes. The document is WIP.
 
-- Input: TiDB clusters.
-- Output: MySQL, TiDB, Kafka or incremental backup files.
-- Compatibility: Compatible with TiDB v2.1 and later versions.
-- Kubernetes: Supported. See [TiDB Binlog Cluster Operations](/tidb-in-kubernetes/maintain/tidb-binlog.md) and [TiDB Binlog Drainer Configurations in Kubernetes](/tidb-in-kubernetes/reference/configuration/tidb-drainer.md).
+### Incremental data export tools
 
-## Tools development roadmap
+#### TiDB Binlog
 
-To help you understand the relationships between the above tools, here is a brief introduction to TiDB ecosystem tools development roadmap.
+[TiDB Binlog](/reference/tidb-binlog/overview.md) is a system that collects binlog for TiDB clusters and provides tools for near real-time sync and backup.
 
-### TiDB backup and restore
+The following are the basics of TiDB Binlog:
 
-Mydumper and Loader -> BR:
+- Input/Output:
+    - Input: TiDB Cluster
+    - Output: MySQL, TiDB, Kafka or incremental backup files
+- Supported TiDB versions: v2.1 or later
+- Kubernetes support: Yes. See [TiDB Binlog Cluster Operations](/tidb-in-kubernetes/maintain/tidb-binlog.md) and [TiDB Binlog Drainer Configurations in Kubernetes](/tidb-in-kubernetes/reference/configuration/tidb-drainer.md) for details.
+
+#### CDC (Beta, under development, ETA May/June 2020 with TiDB 4.0)
+
+[CDC](/reference/tools/ticdc/overview.md) (Change Data Capture) is a system that collects changelog for key value pairs in TiKV and outputs to downstream systems in transactional order.
```

Comment:
The English version of CDC documentation is not merged yet so this is a dead link. This PR should be merged after https://github.com/pingcap/docs/pull/2059 merges.

Reply:
ok